objects/9ad187/script.lua

```diff
@@ -278,15 +275,28 @@ function toggleNumPlayers(_, value)
 end
 function updateBoardLayouts(numPlayers)
```

Comment:
should this function just accept `numBoards` instead now?

Reply:
I don't think so. There are a couple of places that call this, and they'd both need to be updated to pass the new value in.